src/scripts/main.js

```diff
@@ -2,7 +2,25 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('tbody');
 
-// write your code here
+people.forEach(element => {
+  const {
+    sex,
+    born,
+    died,
+    age = died - born,
+    century = Math.ceil(died / 100),
+  } = element;
```

Comment:
Your declaration of `age` and `century` seems interesting. But it's can be confused, as for me.
```suggestion
  const {
    sex,
    born,
    died,
  } = element;
  
  const age = died - born;
  const century = Math.ceil(died / 100);
```